src/libs/utils.js

```diff
@@ -1,8 +1,27 @@
 const fs = require('fs-extra');
+const path = require('path');
+const gitUrl = require('parse-github-url');
 
 module.exports = {
   readConfigFile(file) {
+    // try to load package.json from execution context directory
+    const pkg = fs.readJSONSync(path.join(process.cwd(), 'package.json'), { throws: false });
+    let host, repo;
+
+    if (pkg) {
+      const pkgRepo = pkg.repository.url || pkg.repository;
+      const pkgGit = gitUrl(pkgRepo);
+      host = `${pkgGit.host}/api/v3`;
```

Comment:
I dont think, we need to override it. Because normally the API path should be `api.github.com`, unless you setup the host in `.santari.json`.